ClosedXML/Excel/Rows/XLRow.cs

```diff
@@ -336,7 +336,7 @@ public IXLRow AdjustToContents(Int32 startColumn, Int32 endColumn, Double minHei
                         else
                         {
                             Double rotation;
-                            if (textRotation == 90 || textRotation == 180 || textRotation == 255)
+                            if (textRotation == 90 || textRotation == 180)
```

Comment:
Actually, I have no idea why there is a special case for `textRotation == 255` a few lines back.

Reply:
`textRotation == 255` stands for a vertical orientation, so it is natural to determine a height for this case independently.
![image](https://user-images.githubusercontent.com/19576939/38802680-31b0c0f4-417e-11e8-9262-c779dd4ed571.png)
I wonder if this special case was ever tested?..